docs/book/src/tasks/certs/experimental-ca-rotation.md

```diff
@@ -0,0 +1,103 @@
+
+- [ ] a working & stable workload cluster should have 3 secrets provisionned with "old" CAs
+- [ ] backup/copy existing Secret as `[cluster name]-**-backup` to keep old state of the workload cluster & allow recovery
+- [ ] in management cluster : update CA Secret with `pubCA1+2` and `CAkey1`
+- [ ] in management cluster : update PROXY Secret with `pubPROXY1+2` and `PROXYkey1`
+- [ ] in management cluster : update ETCD Secret with `pubETCD1+2` and `ETCDkey1`
+- [ ] ensure kube-controller-manager still signs using old pubCA only with `--cluster-signing-cert-file` (It doesn't support any CA bundle, as explained in [Kubernetes manual CA rotation - official](https://kubernetes.io/docs/tasks/tls/manual-rotation-of-ca-certificates/#rotate-the-ca-certificates-manually) )
+- [ ] edit workload cluster's cluster-info ConfigMap to match `pubCA1+2`
+- [ ] in management cluster : rollout all `MachineDepoyment` worker machines used by the workload cluster (e.g set `RolloutAfter` field or run `clusterctl alpha rollout restart` command)
+- [ ] in management cluster : rollout `KubeadmControlPlane` machines used by the workload cluster (e.g set `RolloutAfter` field or run `clusterctl alpha rollout restart` command)
+- [ ] rollout restart all pods running in workload cluster
+
+<h1>LIVE OPERATIONS</h1>
+
+<aside class="note warning">
+
+<h1>Machine remote access is required</h1>
+
+In order to manually replace CA and renew all control-plane certificates at once, this procedure requires logging into systems using SSH. So it breaks CAPI *"no direct access to the machines"* concept and (by nature) will never be generalized nor automated "as it is" by any CAPI controller.
+
+This is only a manual workaround and may not work as expected on all systems. 
+
+</aside>
+
+- [ ] **pause CAPI cluster reconciliation**
+- [ ] replace key & crt files on all control planes & renew cert **on all control-plane nodes at the same time**:
+
+```bash
+# as root
+# kube CA
+echo -n "<new CA private key content>" > /etc/kubernetes/pki/ca.key
+chmod 0600 /etc/kubernetes/pki/ca.key
+
+echo -n "<new CA pubcert + old CA pubcert content>" > /etc/kubernetes/pki/ca.crt
+chmod 0644 /etc/kubernetes/pki/ca.crt
+
+echo -n "<new CA public key content>" > /etc/kubernetes/pki/cluster-signing-ca.crt
+chmod 0644 /etc/kubernetes/pki/cluster-signing-ca.crt
+
+# kube PROXY
+echo -n "<new PROXY private key content>" > /etc/kubernetes/pki/front-proxy-ca.key
+chmod 0600 /etc/kubernetes/pki/front-proxy-ca.key
+
+echo -n "<new PROXY pubcert + old PROXY pubcert content>" > /etc/kubernetes/pki/front-proxy-ca.crt
+chmod 0644 /etc/kubernetes/pki/front-proxy-ca.crt
+
+# kube ETCD
+echo -n "<new ETCD private key content>" > /etc/kubernetes/pki/etcd/ca.key
+chmod 0600 /etc/kubernetes/pki/etcd/ca.key
+
+echo -n "<new ETCD pubcert + old ETCD pubcert content>" > /etc/kubernetes/pki/etcd/ca.crt
+chmod 0644 /etc/kubernetes/pki/etcd/ca.crt
+
+# regen certs on all VM at the same time !
```

Comment:
Not sure about this comment, `kubeadm certs renew all` only works on the machine where this command is executed